core/shared/src/main/scala/cats/parse/Parser.scala

```diff
@@ -1407,41 +1408,43 @@ object Parser extends ParserInstances {
 
     }
 
-  private object Impl {
+  private[parse] object Impl {
```

Comment:
can we revert this to just private?

Reply:
No, it's a member of `Parser`, and is accessed in `Parser0` in `parse`, `parseAll`, `?` and `parseMut`.